cookbook/doctrine/registration_form.rst

```diff
@@ -19,12 +19,12 @@ first start with :doc:`/cookbook/security/entity_provider`.
 
 Your ``User`` entity will probably at least have the following fields:
 
-``username``
+* ``username``
```

Comment:
If you browse this page (http://symfony.com/doc/current/cookbook/doctrine/registration_form.html) you'll understand why I did this :)

![symfony_doc_error](https://cloud.githubusercontent.com/assets/73419/10558222/c0aed5dc-74c8-11e5-85ac-2313ca3a19bb.png)